config.yaml.example

```diff
@@ -10,6 +10,17 @@ plugins:
     rabbitmq:
       host: localhost
       port: 5672
+      username: guest
+      password: guest
+      vhost: /
+      naming_join_pattern: . # symbol to concatenate names with. Example queue-name: threatbus.intel."hostname"
+      queue:
+        name_suffix: "my_suffix" # optional. remove property / set empty to use 'hostname'
+        durable: true
+        auto_delete: false
+        lazy: true
+        exclusive: false
+        max_items: 100000 # optional. remove property / set to 0 to allow infinite length
```

Comment:
Unless these are clear to every RabbitMQ user, it would be nice to see one comment per option. For example, I wouldn't know what `auto_delete` or `exclusive` does.

Reply:
These [queue properties](https://www.rabbitmq.com/queues.html#properties) should be clear for every RabbitMQ user.